src/views/pages/Login/index.vue

```diff
@@ -24,17 +24,24 @@ import Vue from 'vue';
 import TokenService from '@/services/Token';
 import CoreAPI from '@/utils/CoreAPI';
 import SyncService from '@/services/Sync';
+import { MutationType as UserMutation } from '@/store/modules/user/types';
 
 export default Vue.extend({
   name: 'Login',
   mixins: [TokenService.tokenMixin()],
   methods: {
     async login() {
       try {
-        await CoreAPI.login(this.credentials);
+        const { uuid, jwt, type } = await CoreAPI.login(this.credentials);
+        this.$store.commit(UserMutation.SET_USER_ID, { userId: uuid });
+        this.$store.commit(UserMutation.SET_USER_TYPE, { type });
         this.$router.push({ name: 'Home' });
       } catch (err) {
-        this.error = 'Invalid username or password';
+        if (err.message  === 'Unauthorized user') {
```

Comment:
I think checking the type of an error should be done with a more permanent property, as the message is prone to change

Reply:
updated!